src/index.html

```diff
@@ -8,22 +8,9 @@
         <link rel="stylesheet" href="./index.css">
         <link rel="shortcut icon" href="#">
         <meta name="description" content="SuperSplat is an advanced browser-based editor for manipulating and optimizing 3D Gaussian Splats. It is open source and engine agnostic." />
-        <meta name="keywords" content="PLY, Gaussian, Splat, PlayCanvas, Editor" />
         <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0" />
 
-        <!-- Open Graph / Facebook -->
-        <meta property="og:type" content="website" />
-        <meta property="og:url" content="https://playcanvas.com/supersplat/editor" />
-        <meta property="og:title" content="SuperSplat" />
-        <meta property="og:description" content="SuperSplat is an advanced browser-based editor for manipulating and optimizing 3D Gaussian Splats. It is open source and engine agnostic." />
-        <meta property="og:image" content="https://playcanvas.com/supersplat/editor/static/images/header.webp" />
-
-        <!-- Twitter -->
-        <meta property="twitter:card" content="summary_large_image" />
-        <meta property="twitter:url" content="https://playcanvas.com/supersplat/editor" />
-        <meta property="twitter:title" content="SuperSplat" />
-        <meta property="twitter:description" content="SuperSplat is an advanced browser-based editor for manipulating and optimizing 3D Gaussian Splats. It is open source and engine agnostic." />
-        <meta property="twitter:image" content="https://playcanvas.com/supersplat/editor/static/images/header.webp" />
+        <!-- ssr-head -->
```

Comment:
I suspect you can inject meta tags in the head element without needing a string to match. You have the `</head>` for example.

Reply:
One little concern I had is that the order might be important. If I place at the end, it would be after the `<script>` tags and that might not be SEO friendly. 

I'll still place it at the end of <head> anyways and we'll see how it behaves.